lib/offline/OfflineStorage_SQLite.cpp

```diff
@@ -755,8 +755,17 @@ namespace MAT_NS_BEGIN {
             if (!stmt.select() || !stmt.getRow(m_pageSize)) { return false; }
         }
 
+#if defined(_MSC_VER)
 #pragma warning(push)
 #pragma warning(disable:4296) // expression always false.
```

Comment:
Why does the code want to compare uintxx_t x < 0? Just curious why this is happening.

Reply:
No idea, seems like it's unhittable code, but not something I want to address in this PR.